doc/python/cone-plot.md

```diff
@@ -126,5 +126,43 @@ fig.update_layout(scene=dict(aspectratio=dict(x=1, y=1, z=0.8),
 fig.show()
 ```
 
+### Sizemode
+
+Earlier examples use `sizemode="absolute"` when adjusting the cone size scaling with `sizeref`. `sizemode` also supports `raw`(new in 5.21) and `scaled`.
+
+```python
+import plotly.graph_objects as go
+import pandas as pd
+
+df = pd.read_csv(
+    "https://raw.githubusercontent.com/plotly/datasets/master/cone_plot_data.csv"
```

Comment:
Non-blocking
You may rename this dataset to something like `wind_data` and later reuse it to make `streamtube`, `volume`, `isosurface`, etc.